examples/python/l1-resources/project.py

```diff
@@ -0,0 +1,24 @@
+from aws_cdk import (
+  # Duration,
```

Comment:
[q] do we need these commented values in the import?

Reply:
removed